map/map_tests/ge0_parser_tests.cpp

```diff
@@ -223,7 +221,7 @@ UNIT_TEST(LatLonFullAndClippedCoordinates)
     }
   }
 
-  for (int coordSize = 1; coordSize <= 8; ++coordSize)
+  for (size_t coordSize = 1; coordSize <= 8; ++coordSize)
```

Comment:
uint8_t ?

Reply:
No, why?
uint8_t is still too big, so I prefer to stick to a more common type.